libcaf_core/caf/flow/string.hpp

```diff
@@ -85,6 +86,48 @@ class to_lines_step {
   std::vector<char> buf_;
 };
 
+/// Turns a sequence of bytes into a sequence of chunks.
+class to_chunks_step {
```

Comment:
This has no connections to strings. Move to e.g. `libcaf_core/caf/flow/chunk.hpp`?

Reply:
What do you think of renaming the file instead ? From 'flow/string.hpp' to something like 'flow/transformations.hpp'?